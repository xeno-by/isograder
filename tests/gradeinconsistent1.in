* Etape 1 [/20]

** Classe Point [/2]
** Classe Vector [/3]
