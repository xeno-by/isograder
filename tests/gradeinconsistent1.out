EXIT CODE:
1

STDOUT:

STDERR:
error parsing the correction sheet:
line 1: invalid section grade: expected (i.e. specified) 20, actual (i.e. sum of all subsection grades) 5
* Etape 1 [/20]
^

